modules/module-headings.php

```diff
@@ -192,7 +192,7 @@ function jetpack_get_module_i18n( $key ) {
 			),
 
 			'sso' => array(
-				'name' => _x( 'Single Sign On', 'Module Name', 'jetpack' ),
+				'name' => _x( 'Secure Sign On', 'Module Name', 'jetpack' ),
```

Comment:
We try not to edit this file directly; it is automatically generated when building the plugin. Since you changed the module heading in `modules/sso.php`, you've already done what is needed to update this value 👍 	Could you remove that change form your PR?

Thank you!

Reply:
Ok reverted the change. Thanks for pointing that out.